araproc/analysis/standard_reco.py

```diff
@@ -258,108 +270,78 @@ def do_standard_reco(self, wavepacket):
         if(self.__latest_event_num != event_number):
           self.__latest_event_num = event_number
 
-          # get the correlation functions
-          self.__corr_functions_v = self.__calculate_cross_correlations(waveform_bundle, self.pairs_v)
-          self.__corr_functions_h = self.__calculate_cross_correlations(waveform_bundle, self.pairs_h) 
+        # get the correlation functions
+        self.__corr_functions_v = self.__calculate_cross_correlations(waveform_bundle, self.pairs_v)
+        self.__corr_functions_h = self.__calculate_cross_correlations(waveform_bundle, self.pairs_h) 
         
+        ############################
+        ####### VPol Pulser ########
+        ############################
+
         # check the cal pulser in V
         pulser_map_v = self.rtc_wrapper.correlators["nearby"].GetInterferometricMap(
-            self.pairs_v,
-            self.__corr_functions_v,
-            0
-        )
+            self.pairs_v, self.__corr_functions_v, self.__arrival_delays_v, 0,)
```

Comment:
Idk if this is a comment or curiosity, but is the hanging comma important?

Reply:
It's the latest recommendation from e.g. ruff and flake8: https://docs.astral.sh/ruff/rules/missing-trailing-comma/

Reply:
Ohh very interesting! Never thought about that. Thanks!